Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -358,6 +359,16 @@ public enum DefaultFunctionOpacityOptions {
     @"
 If verification fails, report a detailed counterexample for the first failing assertion (experimental).".TrimStart()) {
   };
+  
+  public static readonly Option<IList<FileInfo>> TranslationRecords = new("--translation-record", 
```

Comment:
I regret the existence of `CommonOptionBag` :-D

Any chance we could move these options to more specific locations? I would prefer `ExecutableBackend`.

Reply:
Moved them to `IExecutableBackend`, right next to `OuterModule` :)